docs/synapse/userguides/storm_ref_filter.rstorm

```diff
@@ -966,25 +1106,27 @@ Despite its similarity to a pivot operation, embedded property syntax is commonl
 *<query>* [ **+ | -** ] **:** *<prop>* **::** *<prop>* **=** *<pval>*
 
 .. NOTE::
-  In Storm, the leading colon (i.e., the colon before the name of the initial secondary property) is **required**.
-  When using this syntax to create an embed column in Optic, the initial colon should be **omitted** (i.e.,
-  ``asn::name`` vs ``:asn::name``). Optic will prepend the initial colon for you.
+  
+  In Storm, the leading colon (i.e., the colon before the name of the initial secondary property) is
+  **required**. When using this syntax to create an embed column in Optic, the initial colon should be
+  **omitted** (i.e., ``asn::name`` vs ``:asn::name``). Optic will prepend the initial colon for you.
 
 
 **Examples:**
 
 
 *Filter Example - Single Pivot*
 
-- From an initial set of IPv4 addresses, return only those IPv4s registered to an Autonomous System (AS) whose name 
-  starts with "makonix":
+- From an initial set of IPv4 addresses, filter results to only those IPv4s registered to an Autonomous
+  System (AS) whose name starts with "makonix":
 
 .. storm-pre:: [ (inet:ipv4=185.86.150.67 :asn=52173) (inet:asn=52173 :name="makonix, lv") ] inet:ipv4 +:asn::name^="makonix"
```

Comment:
is that extra `inet:ipv4` lift supposed to be a filter?

Reply:
Just a derp